src/CalcViewModel/Common/KeyboardShortcutManager.cpp

```diff
@@ -71,7 +71,7 @@ namespace CalculatorApp
             }
```

Comment:
the 1+1=3 bug is still reproing, looks like it was caused by a bad merge in #585. To fix we will want to remove the extra call to RunFirstEnabledButtonCommand(buttons) on line 574.